frontend/src/components/ScheduledEventsTable/ScheduledEventsTable.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Table } from 'react-bootstrap';
+import { ScheduledEventsTableRow } from '../ScheduledEventsTableRow/ScheduledEventsTableRow';
```

Comment:
Export `ScheduledEventsTableRow` via the index file so you don't have this `'../ScheduledEventsTableRow/ScheduledEventsTableRow'`